test/algorithms/multi_length.cpp

```diff
@@ -8,15 +8,8 @@
 
 #include <algorithms/test_length.hpp>
 
-#include <boost/geometry/multi/multi.hpp>
-
 #include <boost/geometry/geometries/geometries.hpp>
 #include <boost/geometry/geometries/point_xy.hpp>
-#include <boost/geometry/multi/geometries/multi_linestring.hpp>
-
-#include <boost/geometry/io/wkt/wkt.hpp>
```

Comment:
`io/wkt/read.hpp` was included in `test_length.hpp`.
So what I did was to replace `io/wkt/read.hpp` by `io/wkt/wkt.hpp` in test_length.hpp` and removed it from this file.
See also my other comment on the `multi_simplify.cpp` unit test.